docs/examples/Tutorial/GuestFormWithErrorFields.js

```diff
@@ -1,16 +1,16 @@
 import React from 'react';
-
-import GuestSchema from './GuestSchema';
 import {
   AutoForm,
```

Comment:
What about the order of imports here?

Reply:
After rendering in `ExampleCustomizer` all `universal` imports are renamed to the correct theme (`uniforms-semantic`, etc.). I wanted to keep the correct order on the rendered docs more than in the code.

Reply:
Alright :)